api/libproject/devstart.rb

```diff
@@ -1348,3 +1348,25 @@ def setup_cloud_project(cmd_name, *args)
   :description => "Initializes resources within a cloud project that has already been created",
   :fn => lambda { |*args| setup_cloud_project("setup-cloud-project", *args) }
 })
+
+def start_api_and_incremental_build(cmd_name, args)
+  ensure_docker cmd_name, args
+  common = Common.new
+  begin
+    # appengineStart must be run with the Gradle daemon or it will stop outputting logs as soon as
+    # the application has finished starting.
+    common.run_inline %W{gradle --daemon appengineStart}
+    # incrementalHotSwap must be run without the Gradle daemon or stdout and stderr will not appear
+    # in the output.
+    common.run_inline %W{gradle --no-daemon --continuous incrementalHotSwap}
```

Comment:
So this blocks? Are logs shown in the console window at this point?

Reply:
Not the server logs. This blocks on the compile and hot-swap, so those logs will show (compile errors too), but the server logs are only available by tailing the file mentioned when the server starts. Actually, check out my latest commit. I added a tail so the logs appear here too.